action.yaml

```diff
@@ -2,6 +2,10 @@
 name: Terraform
 description: Terraform all-in-one workflow
 inputs:
+  detailed_exit_code:
+    required: false
+    description: Whether to report a detailed exit code
+    default: 'false'
```

Comment:
This name doesn't have any meaning at this level. It reflects the parameter used by the new feature. But it doesn't indicate what the impact would be.

Perhaps better to add something like a `check for changes` in addition to `plan` and `apply`?

Reply:
I'm okay with that. Should we even have it be part of `apply` then? If changes are going to be automatically applied, having a detailed exit code probably doesn't add any value at all. The job could then skip `apply` and `destroy` before reporting final status.

Reply:
It kinda depends on which use-case(s) you're trying to solve here. I can think of several, each possibly requiring different solutions :wink:

Reply:
And looking at the PR this one's mentioned in, perhaps the actual reporting to Slack could be made part of this actions as well? :shrug: